public/content/en/welcome/welcome-to-d.md

```diff
@@ -2,39 +2,31 @@
 
 Welcome to the online tour of the *D Programming language*.
 
-This tour will give you an overview of this powerful and expressive systems
-programming language which compiles directly to efficient, *native*
-machine code.
-
-### What is D
-
-D has many [unique features](http://dlang.org/overview.html) and is a
-general purpose systems and applications programming language.
-It is a _high level language_, but retains the ability to write _high
-performance_ code and interface directly with the _operating system API's_
-and with hardware.
-D is well suited to writing medium to large-scale programs.
-D is _easy to learn_, provides many capabilities to aid the programmer,
-and is well suited to aggressive compiler optimization technology.
-D is not a scripting language, nor an interpreted language.
-It doesn't come with a VM, a religion, or an overriding philosophy.
-It's a _practical language for practical programmers_ who need to get the
-job done _quickly_, _reliably_, and leave behind _maintainable_,
-_easy to understand_ code.
+This tour will give you an overview of this __powerful__ and __expressive__
+language which compiles directly to __efficient__, __native__ machine code.
+
+### What is D?
 
 D is the culmination of _decades of experience implementing compilers_
-for many diverse languages,
-and attempting to construct large projects using those languages.
-D draws inspiration from those other languages (most especially C++)
-and tempers it with experience and real world practicality.
+for many diverse languages and has a large number of
+[unique features](http://dlang.org/overview.html):
+
+- _high level language_ with great modeling power
```

Comment:
And: Should we maybe emphasise that D also suits well for functional programming? Something like: "D allows writing code in an imperative, object-oriented or functional style"?